player/command.c

```diff
@@ -7594,6 +7602,18 @@ static void command_event(struct MPContext *mpctx, int event, void *arg)
         run_command_opts(mpctx);
     }
 
+    if (event == MPV_EVENT_VIDEO_RECONFIG && ctx->hwdec_osd_mode) {
+        struct track *track = mpctx->current_track[0][STREAM_VIDEO];
+        struct mp_decoder_wrapper *dec = track ? track->dec : NULL;
+        char *current = NULL;
+        if (dec)
+            mp_decoder_wrapper_control(dec, VDCTRL_GET_HWDEC, &current);
+        if (!current || !current[0])
+            current = "no";
+        show_property_osd(mpctx, "hwdec", ctx->hwdec_osd_mode);
+        ctx->hwdec_osd_mode = 0;
+    }
```

Comment:
Can we unify this with the other hwdec notification? https://github.com/mpv-player/mpv/blob/2542a7824c11f7eedae1d6232c5d136722b31b97/video/decode/vd_lavc.c#L1323-L1331

Reply:
Do you mean make the message the same? I think this is too long to show on the OSD personally.

Reply:
No, not that. Can we handle both of those messages in one place instead of having it split in two places?

Reply:
We technically could but I would prefer not to. The vd_lavc message being from a different module has meaning.

Reply:
Ok.